js/okcoin.js

```diff
@@ -199,6 +199,7 @@ module.exports = class okcoin extends Exchange {
                     },
                 },
                 'margin': {
+                    // Margin trading closed down on February 21, 2022
                     'get': {
```

Comment:
why are we keeping margin endpoints? are they still valid?

Reply:
> why are we keeping margin endpoints? are they still valid?

I think some of them still work for people who placed margin orders before it was shut down and I think there's a chance they could bring margin trading back, from some of their blog posts it looks like in the past they've closed down margin trading and brought it back. We can delete them if you think that's necessary though

Reply:
@Dan-krm agreed let's keep them for now